bazel/http_archives.bzl

```diff
@@ -26,27 +26,34 @@ def register_http_archive_dependencies():
 
     http_archive(
         name = "com_google_protobuf",
-        sha256 = "dc167b7d23ec0d6e4a3d4eae1798de6c8d162e69fa136d39753aaeb7a6e1289d",
-        strip_prefix = "protobuf-23.1",
-        url = "https://github.com/protocolbuffers/protobuf/archive/refs/tags/v23.1.tar.gz",
+        sha256 = "8ff511a64fc46ee792d3fe49a5a1bcad6f7dc50dfbba5a28b0e5b979c17f9871",
+        strip_prefix = "protobuf-25.2",
```

Comment:
25.2 is the oldest release that contains a needed apple_common [fix](https://github.com/protocolbuffers/protobuf/pull/15040) (not in the release notes 😢)